cmd/demo/demo.go

```diff
@@ -101,7 +108,7 @@ func main() {
 		isTable, tableRendered := false, false
 		table := tablewriter.NewWriter(os.Stdout)
 
-		stream := sql.Run(slct, db, "")
+		stream := sql.Run(slct, db, *modelDir)
```

Comment:
Is it a better idea to create the modelDir in `sql.Run` than here in `main`? I vaguely feel that it is easier to maintain the code if we allocate the resource right before we use it.

Reply:
Good idea.
SQLFlow needs a model directory only a SQL is extended.